packages/aws-cdk/test/integ/cli/test-cdk-deploy-nested-stack-with-parameters.sh

```diff
@@ -6,7 +6,7 @@ source ${scriptdir}/common.bash
 
 setup
 
-stack_arn=$(cdk deploy -v ${STACK_NAME_PREFIX}-with-nested-stack-using-parameters --parameters "MyTopicParam=ThereIsNoSpoon")
```

Comment:
The `STACK_NAME_PREFIX` was added so that we can run multiple instances of these tests. Otherwise they are attempting to create the same SNS topic.

Reply:
Thanks for the comment. Should we also add to comment in the code?

Reply:
i'll add